packages/core/test/limit-test.js

```diff
@@ -65,6 +69,16 @@ describe('debounce', function () {
     })
   })
 
+  it('should not repeat last event when end is delayed', () => {
+    // See https://github.com/cujojs/most/issues/514
+    var expected = Math.random()
+    var end = delay(5, now(null))
+    var debounced = debounce(1, until(end, startWith(expected, never())))
+
+    return collectEventsFor(6, debounced)
```

Comment:
Why 6 ticks?

Reply:
Lol, I think I was playing around with the test, and forgot to change it back to 5.

Also, `var` 😲

Reply:
See ce1067b

Reply:
I didn't even notice var 😮